torchgeo/datamodules/inria.py

```diff
@@ -0,0 +1,236 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""InriaAerialImageLabeling datamodule."""
+
+from typing import Any, Dict, List, Optional, Tuple, Union, cast
+
+import kornia.augmentation as K
+import pytorch_lightning as pl
+import torch
+import torchvision.transforms as T
+from einops import rearrange
+from kornia.contrib import compute_padding, extract_tensor_patches
+from torch.nn.modules.utils import _pair
+from torch.utils.data import DataLoader, Dataset
+from torch.utils.data._utils.collate import default_collate
+
+from torchgeo.datamodules.utils import dataset_split
+from torchgeo.datasets import InriaAerialImageLabeling
```

Comment:
```suggestion
from .utils import dataset_split
from ..datasets import InriaAerialImageLabeling
```
For stuff in `torchgeo/`, we should always use local imports to avoid confusion if someone puts a file or directory in their `PYTHONPATH` or current directory with a similar name. For stuff in `tests/`, we should always use `from torchgeo` imports so we can test the installed copy instead of the local copy. Btw, can you also change this in `datasets/inria.py`? That's the only other place that makes this mistake.

Reply:
Done